test/fixtures/project/script/commands/enable_config_file.yml

```diff
@@ -0,0 +1,2 @@
+key1: "value1"
+key2: "value2"
```

Comment:
This file was probably generated before we added FakeFS. I think we can 🔥 this and it shouldn't regenerate